main/src/ca/uwaterloo/flix/language/phase/ClosureConv.scala

```diff
@@ -686,6 +692,10 @@ object ClosureConv {
         val e = visitExp(exp)
         Expression.Force(e, tpe, loc)
 
+      case Expression.Instanceof(exp, clsName, loc) =>
```

Comment:
The informal convention is:

- If the type is string (i.e. early in the pipeline) it is `className`
- Once the `className` becomes a `Class[T]` object we call it `clazz`. 

(You can look at e.g. `NewObject`)

I believe this change happens between NamedAst and ResolvedAst (i.e. in the Resolver).

Reply:
Thus after `ResolvedAst` it should be `clazz`.

Reply:
All names should be changed, I've checked with grep as well.